app/assets/javascripts/discourse/app/components/sidebar.js

```diff
@@ -0,0 +1,3 @@
+import GlimmerComponent from "discourse/components/glimmer";
+
+export default class Sidebar extends GlimmerComponent {}
```

Comment:
Did you know this component is empty!?! :P (jk I know it is intentionally empty)